pkg/kubelet/pluginmanager/cache/types.go

```diff
@@ -20,28 +20,31 @@ package cache
 // order to consume plugins
 // The PluginHandler follows the simple following state machine:
 //
-//                         +--------------------------------------+
-//                         |            ReRegistration            |
-//                         | Socket created with same plugin name |
-//                         |                                      |
-//                         |                                      |
-//    Socket Created       v                                      +        Socket Deleted
+//	                     +--------------------------------------+
+//	                     |            ReRegistration            |
+//	                     | Socket created with same plugin name |
+//	                     |                                      |
+//	                     |                                      |
+//	Socket Created       v                                      +        Socket Deleted
+//
 // +------------------> Validate +---------------------------> Register +------------------> DeRegister
-//                         +                                      +                              +
-//                         |                                      |                              |
-//                         | Error                                | Error                        |
-//                         |                                      |                              |
-//                         v                                      v                              v
-//                        Out                                    Out                            Out
+//   - +                              +
+//     |                                      |                              |
+//     | Error                                | Error                        |
+//     |                                      |                              |
+//     v                                      v                              v
+//     Out                                    Out                            Out
```

Comment:
oof... gofmt is not kind to ascii art

Reply:
fixing. i am not quite sure how this got tripped up.